data/events/scripts/creature.lua

```diff
@@ -1,14 +1,15 @@
 function Creature:onChangeOutfit(outfit)
-	return true
+	if hasEventCallback(EVENT_CALLBACK_ONCHANGEOUTFIT) then return EventCallback(EVENT_CALLBACK_ONCHANGEOUTFIT, self, outfit) else return true end
```

Comment:
no one liners for better readability, here and in the two bellow

Reply:
+1 this look ugly as hell

Reply:
I actually did this intentional, so people don't start screwing around with it (because usually people don't touch one liners) but I can change it everywhere I don't mind.